saleor/dashboard/order/forms.py

```diff
@@ -614,6 +614,16 @@ def __init__(self, *args, **kwargs):
         for form in self.forms:
             form.empty_permitted = False
 
+    def clean(self):
+        if any(self.errors):
+            return
+
+        total_quantities = sum(
+            form.cleaned_data['quantity'] for form in self.forms)
+        if total_quantities <= 0:
```

Comment:
I'm not 100% if we need this validation. If we didn't have it, fulfilling 0 quantities for all lines would redirect the user to order details view with a message "No lines fulfilled" which quite makes sense.

Reply:
But that gives wrong impression to users that it looks like the fulfillment is succeeded. Actually it has errors and the current form should reflect the errors. The user would like to fulfill the order indeed and so we shouldn't redirect to other pages if the form has errors.

Reply:
OK, let's leave this validation then 👍